server/src/main/java/org/apache/cassandra/sidecar/tasks/CdcRawDirectorySpaceCleaner.java

```diff
@@ -182,10 +182,22 @@ protected void routineCleanUp()
     {
         for (InstanceMetadata instanceMetadata : instancesMetadata.instances())
         {
-            List<String> dataDirectories = instanceMetadata.dataDirs();
-            dataDirectories.stream()
-                           .map(dir -> new File(dir, CDC_DIR_NAME))
-                           .forEach(this::cleanUpCdcRawDirectory);
+            String cdcDir = instanceMetadata.cdcDir();
+            if (cdcDir != null)
+            {
+                try
+                {
+                    cleanUpCdcRawDirectory(new File(cdcDir));
+                }
+                catch (Exception e)
+                {
+                    LOGGER.warn("Unable to clean up CDC directory {}", cdcDir, e);
```

Comment:
apologies, I think we need to log the instance in the message too:
```suggestion
                    LOGGER.warn("Unable to clean up CDC directory {} for instance {}", cdcDir, instanceMetadata, e);
```